pyanaconda/modules/payloads/payload/rpm_ostree/installation.py

```diff
@@ -50,11 +50,12 @@ def safe_exec_with_redirect(cmd, argv, successful_return_codes=(0,), **kwargs):
 
     :raise: PayloadInstallationError if the call fails for any reason
     """
-    rc = execWithRedirect(cmd, argv, **kwargs)
+    rc, output = execProgram([cmd] + argv, **kwargs)
```

Comment:
Seems you would be fine to use ' execWithCapture ' and let the _run_program be internal function to the 'pyanaconda/core/utils.py' file.

Reply:
@cgwalters Do you want me to take over actually the PR? I dont want do bother you with another iteration.

Reply:
I'd love that yes